### PR TITLE
New feature/ enhancement: Images for user engagement

### PR DESCRIPTION
**Pull request by AdiChat**
Adds an image under the search bar at the homepage for _engagement of our audience_. 😃 

This needs to be developed further and this pull request acts as the basic addition.

Further enhancements within this path:
* Need to expand the image set used (currently, ten images)
* Alternative to randomly loading images
* Add credit back to source (possibly, written within the image at bottom corner)

### CODE REVIEW

File: cosmos_search/settings.py [hunk 1]
```diff
@@ -23,7 +23,7 @@
 SECRET_KEY = 'groe@_2fh_a68s=kbjdb1$zy3jut%2%*)cg^ji0qq_*&a*d-r_'
 
 # SECURITY WARNING: don't run with debug turned on in production!
-DEBUG = False
+DEBUG = True
```
Comment (vaibhavsingh97) [inline cosmos_search/settings.py]: `DEBUG` should be false

Comment (AdiChat) [reply cosmos_search/settings.py]: Yes 👍

**Comment by AdiChat** [comment]
It looks like I did not set `url.py` and the error came up on changing `debug` to `false`. I will fix it soon.

**Comment by AdiChat** [comment]
Though `staticfiles` app is inefficient and insecure, I believe this will not harm the usage of our app considering the alternatives. 

The app runs on setting the insecure option: `python manage.py runserver --insecure`

This will force serving the static files through the `staticfiles` app even when `debug` is `False`.

**Comment by AdiChat** [comment]
Got rid of `staticfiles` app and serving images through [`data-store`](https://github.com/OpenGenus/data-store)
Everyone must be happy now 😄

**Comment by Pl4gue** [comment]
Do we want to leave this open for further development?

**Comment by AdiChat** [comment]
We shall merge this as this feature, in the current state, looks fine. 
We will develop over this feature in separate pull requests.

**Comment by AdiChat** [comment]
I am merging this pull request 👍

**Pull request merged**